### DISCUSSION TITLE
Music effect background

### DISCUSSION THREAD

**Discussion by 0rsa** [awawa-dev/HyperHDR#626] [General]
Good morning everyone.

I have something strange to expose.
I had to reinstall my OS yesterday and I had also reinstall hyperHDR, the latest stable version (I was using the same stable version before the reinstallation)

I use especially the music background effect called "Music: equalizer test (turn on video preview)" because I mounted one piece of leds on each speaker.

With the fresh Hyperion's installation, I cannot retrieve the same effect I was having before.

The old effect :

https://github.com/awawa-dev/HyperHDR/assets/769750/80f20370-073b-4f64-a8e5-3dde38b7f470

The audio vizualizer work with two dimensions. The width of the bar varies with the audio level and the colors change according to the sound frequencies. It was really good.

The same new effect with the fresh installation (Music: equalizer test (turn on video preview) :

https://github.com/awawa-dev/HyperHDR/assets/769750/e179cfe6-c46d-412c-b200-e7916adccac9

The width of the audio visualizer is now fixed (red, low frequency at the right, blue high frequency at the left) and the intensity of the colour is determined by the current frequencies.

I would like to retrieve the old effect that I appreciate so much.
Do you have an idea of what happened between the two installations of the same version?

Thanks, have a nice day.

**Comment by awawa-dev** [comment]
Hi
Most likely you are using now different effect than before: 'equalizer' which is meant for testing purpose only. Check video live preview (with video stream and LEDs visualisations enabled) in HyperHDR and logs.

**Comment by 0rsa** [comment]
Thank you for your reply.
Do you know what is the effect name which can accomplish the effect seen in the first video?

**Comment by 0rsa** [comment]
Ok I finally found the trick: this is related to the LED layout.
Playing with the classic layout, the number of leds top / bottom / left / right and the input position, you can accomplish what I've done in the first video.
I confirm that you need the effect "Music: equalizer test" to get this render.

**Comment by awawa-dev** [comment]
Most music effects support the geometry of the LED strip, so yes, it must be configured first according to what you see in the live view (most music effects only affect selected areas). It is also recommended to back up your HyperHDR configuration at least once: it's just a few mouse clicks. BTW very nice LEDs mod for speakers 👍
